acceptance/bundle/environments/dependencies/script

```diff
@@ -0,0 +1,5 @@
+trace $CLI bundle validate
+
+trace $CLI bundle deploy
+trace jq -s '.[] | select(.path=="/api/2.2/jobs/create") | .body.environments' out.requests.txt
```

Comment:
Q: do you assert on the request payload because it includes the expanded wheel path?

Reply:
Yes, exactly